packages/helloworld/ios/HelloWorld/AppDelegate.swift

```diff
@@ -10,19 +10,49 @@ import React_RCTAppDelegate
 import ReactAppDependencyProvider
 import UIKit
 
+
 @main
-class AppDelegate: RCTAppDelegate {
-  override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
-    self.moduleName = "HelloWorld"
-    self.dependencyProvider = RCTAppDependencyProvider()
+class AppDelegate: UIResponder, UIApplicationDelegate, UIWindowSceneDelegate {
+  var window: UIWindow?
+  
+  var reactNativeFactory: RCTReactNativeFactory!
+  var reactNativeDelegate: ReactNativeDelegate!
```

Comment:
our swift lint rule don't really like implicitly unwrapped optionals. Can we add a couple of `// swiftlint:disable:next` comments?